--- a/pythonlib.cmake	2020-09-16 16:05:21.000000000 -0700
+++ b/pythonlib.cmake	2020-09-16 08:58:18.000000000 -0700
@@ -132,13 +132,26 @@
 # set_target_properties(ocl PROPERTIES VERSION ${MY_VERSION})
 # endif (WIN32)
 
+if (APPLE AND NOT CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 install(
   TARGETS ocl
-  LIBRARY DESTINATION ${PYTHON_ARCH_PACKAGES}
+  LIBRARY DESTINATION lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/ocl
 )
 # these are the python helper lib-files such as camvtk.py 
 install(
   DIRECTORY lib/
+  DESTINATION lib/python${Python3_VERSION_MAJOR}.${Python3_VERSION_MINOR}/site-packages/ocl
+  #    PATTERN .svn EXCLUDE
+)
+else()
+install(
+  TARGETS ocl
+  LIBRARY DESTINATION ${PYTHON_ARCH_PACKAGES}
+)
+# these are the python helper lib-files such as camvtk.py
+install(
+  DIRECTORY lib/
   DESTINATION ${PYTHON_SITE_PACKAGES}
   #    PATTERN .svn EXCLUDE
 )
+endif (APPLE)
